server/graphql/v2/query/collection/TransactionsCollectionQuery.ts

```diff
@@ -24,9 +29,9 @@ const TransactionsCollectionQuery = {
         'Reference of the account assigned to the other side of the transaction (CREDIT -> sender, DEBIT -> recipient). Avoid, favor account instead.',
     },
     account: {
-      type: AccountReferenceInput,
+      type: new GraphQLList(new GraphQLNonNull(AccountReferenceInput)),
```

Comment:
I tested, this is a nonbreaking change because we're passing it as `account: { slug: $slug }` in both frontend and REST service and it's clever enough to just convert that to `account: [{ slug: $slug }]` automatically.